bochs/iodev/usb/usb_common.h

```diff
@@ -67,31 +69,39 @@
 #define USB_DIR_OUT  0
 #define USB_DIR_IN   0x80
 
-#define USB_TYPE_MASK			(0x03 << 5)
+#define USB_TYPE_MASK			  (0x03 << 5)
```

Comment:
is this space intentional ?
now code look misaligned

Reply:
I saw that too. Found out that it is a difference in spaces and tabs used. The previous commit used tabs. When I modified the line (later reverting back to the original value), my compiler automatically changed the tab to spaces. I did not notice, since they still line up if you have two spaces for every tab. I only noticed afterwards with the submitted diff's. I always use spaces instead of tabs (just preference).

Reply:
I have fixed, but will take a new PR since I have since made other changes to this file.